app/javascript/App/containers/coinSearch/sagas.js

```diff
@@ -10,8 +10,8 @@ export default function* watcher() {
 function* searchCoins({ searchText, searchOpts, namespace }) {
   if (searchText.length < 2) return
   yield apiSagas.get(
-    '/coins.json',
+    '/coins/search.json',
```

Comment:
awesome! much better to use search, instead of just coins.

Reply:
Yeah, the performance for the API Coins Index route was abysmal for autocomplete...